libs/external-db-mysql/project.json

```diff
@@ -32,14 +32,14 @@
         "{options.outputFile}"
       ],
       "options": {
-        "lintFilePatterns": [ "libs/external-db-mysql/**/*.js" ]
+        "lintFilePatterns": [ "libs/external-db-mysql/**/*.ts" ]
       }
     },
     "test": {
       "executor": "@nrwl/jest:jest",
       "outputs": [ "coverage/libs/external-db-mysql" ],
       "options": {
-        "jestConfig": "libs/external-db-mysql/jest.config.js",
+        "jestConfig": "libs/external-db-mysql/jest.config.ts",
```

Comment:
Not sure if it makes sense to rename the file to ts, if the content is written there in js (the config in the file is exported via module.exports)